services/inventory/nodes.go

```diff
@@ -234,3 +227,52 @@ func (s *NodesService) AddRemoteRDSNode(ctx context.Context, req *inventorypb.Ad
 
 	return invNode.(*inventorypb.RemoteRDSNode), nil
 }
+
+// Remove removes Node without any Agents and Services.
```

Comment:
Actually returns error if forse=false and node has services or agents.

Reply:
updated